README.md

```diff
@@ -53,6 +53,11 @@ redirect: https://github.com/<some-user>/<some-repo>/.DEREK.yaml
 If this optional field is non-empty, Derek will read it's configuration from another location. This allows multiple projects to use the same configuration.
 Please note that redirection is only supported for GitHub repository URLs.
 
+* Custom command prefixes
+
+By default, Derek commands can be called with `Derek <some-command>`. The prefix `Derek ` is the default value and this can be changed by using the `DEREKCOMMANDPREFIX` environment variable.
```

Comment:
@neolit123 I vote for `derek_bot_name` or `bot_name` .. @rgee0 wdyt?

Reply:
I think we should go with something from which the user can infer meaning.  Interesting that the issue covering this refers to a `trigger` and this pull request talks of a `command_prefix`.  These are terms that have naturally emerged as we've thought about the nature of the feature.

Derek should still be Derek, whether you shout his name, ring a bell, or make lights flash in order to attract his attention

Reply:
regarding `bot_name`, i should remind that the default value is not only `Derek`, but it's `Derek ` (with an extra space after the name).